.github/workflows/build-docker-api.yml

```diff
@@ -0,0 +1,54 @@
+name: Build a docker image for the API
+
+on:
+  workflow_dispatch:
+    inputs:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Increase watcher limit
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Fetch latest changes
+        run: git fetch --no-tags --prune --depth=5 origin master
+
+      - name: Inject Nx Cloud token
+        shell: bash
+        env:
+          nx_cloud_token: ${{ secrets.NX_CLOUD_TOKEN }}
+        run:
+          sed -i "s/nx_cloud_token/$nx_cloud_token/" $GITHUB_WORKSPACE/nx.json
+
+      - name: Install npm packages
+        run: npm ci
+
+      - name: Build api
+        run: npx nx build api -- --base=origin/master
```

Comment:
you're already installing npm packages, therefore you can use npm run build api no?  no need for npx. also, the double `--` is for passing down arguments when using npm

Reply:
also the base option is unused here. its only used with affected commands to know where to compare from, so you can remove them in both workflows

Reply:
thanks! I didn't know that